WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -571,13 +601,14 @@ import WordPressComAnalytics
                 self?.sharePost(post.objectID, fromView: anchorView)
         })
 
-        if UIDevice.isPad() {
+        if WPDeviceIdentification.isiPad() {
```

Comment:
We're using `UIDevice.isPad()` elsewhere in this class (line 469) – I guess we should standardize and just use one of these helper methods.